config/system.php

```diff
@@ -639,6 +639,13 @@
 'default_value'=>'n'
 );
 
+$system__options_general[]=array(
+'option_name'=>'default_experiment_restriction',
```

Comment:
I would suggest indenting array items for code readability, but it is up to you.

Reply:
In principle I agree. However, the current format of almost all settings in the config/system.php is that it does not include indenting of array items. So I would not apply it here but keep in mind for later.